Procfile

```diff
@@ -0,0 +1 @@
+web: ./carnival
```

Comment:
Ah. That isn't a worker process; it's a binary (`carnival-task`) which gets run daily by Heroku scheduler.